providers/yaml/src/main/java/org/jboss/resteasy/plugins/providers/YamlProvider.java

```diff
@@ -52,10 +53,20 @@
 public class YamlProvider extends AbstractEntityProvider<Object> {
    private static final String ALLOWED_LIST = "resteasy.yaml.deserialization.allowed.list.allowIfBaseType";
    private static final String DISABLE_TYPE_CHECK = "resteasy.yaml.deserialization.disable.type.check";
-   private static final Collection<String> DENY_LIST = Arrays.asList(
-           "javax.script.ScriptEngineManager",
-           URLClassLoader.class.getName(),
-           Object.class.getName()
+   // These types should likely always be allowed
+   private static final Collection<String> DEFAULT_ALLOWED_TYPES = Arrays.asList(
+           BigDecimal.class.getName(),
+           Boolean.class.getName(),
+           Byte.class.getName(),
+           Double.class.getName(),
+           Float.class.getName(),
+           Integer.class.getName(),
+           List.class.getName(),
+           Long.class.getName(),
+           Map.class.getName(),
+           Set.class.getName(),
+           Short.class.getName(),
+           String.class.getName()

```

Comment:
`Character` ...

Reply:
Ah, yes. That is what I missed. Thank you!